MekHQ/src/mekhq/gui/view/PersonViewPanel.java

```diff
@@ -725,6 +729,30 @@ private void fillStats() {
             gridBagConstraints.fill = GridBagConstraints.NONE;
             gridBagConstraints.anchor = GridBagConstraints.NORTHWEST;
             pnlStats.add(lblEdge2, gridBagConstraints);
+            
+            if (person.isAdmin() || person.isDoctor() || person.isTech()) {
```

Comment:
Again, should this be consistent with the other checks? (Support, engineer, etc)

Reply:
This is very intentional. I don't want the person view for Astechs and Medtechs to show available edge that they can't use.  The Ship's Engineer is a composite crewman that doesn't actually have a person view -  individual Vessel Crewmembers are what shows up here, and they're technically combat personnel that shouldn't show the new value either, in my mind.